src/datagrid/DataGridValues.tsx

```diff
@@ -173,9 +191,43 @@ export class GridActions extends React.PureComponent<any, GridActionsState> {
                         alert("Deleted" + selectedRows.length);
                     }}
                 >
-                    DELELTE
+                    DELETE
                 </Button>
             </div>
         );
     }
 }
+
+// Expandable row data
+export const expandableRows = [
+    {
+        content: [
+            {columnName: "User ID", content: 41512},
+            {columnName: "Name", content: "Georgia"},
+            {columnName: "Creation Date", content: "Sep 11, 2008"},
+            {columnName: "Favorite color", content: "Blue"},
+        ],
+        expandableContent:
+            "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin in neque in ante placerat mattis id sed quam. Proin rhoncus lacus et tempor dignissim. Vivamus sem quam, pellentesque aliquet suscipit eget, pellentesque sed arcu. Vivamus in dui lectus. Suspendisse cursus est ac nisl imperdiet viverra. Aenean sagittis nibh lacus, in eleifend urna ultrices et. Mauris porttitor nisi nec velit pharetra porttitor. Vestibulum vulputate sollicitudin dolor ut tincidunt. Phasellus vitae blandit felis. Nullam posuere ipsum tincidunt velit pellentesque rhoncus. Morbi faucibus ut ipsum at malesuada. Nam vestibulum felis sit amet metus finibus hendrerit. Fusce faucibus odio eget ex vulputate rhoncus. Fusce nec aliquam leo, at suscipit diam.",
```

Comment:
We are using same data for all rows, please declare as a variable and reuse.

Reply:
done